### PR TITLE
fix: resolve in-content broken links

### PR DESCRIPTION
**Pull request by katiegoines**
#### Description of changes:
- adjusted page platform lists:
   - added `nextjs` to platform list of all `/[platform]/build-ui` pages
     - [prod](https://docs.amplify.aws/nextjs/build-ui/formbuilder/)
     - [staging](https://correct-content-filtering.d1egzztxsxq9xz.amplifyapp.com/nextjs/build-ui/formbuilder/)
   - removed all but `react`, `javascript`, `nextjs` from `/[platform]/tools/console/tutorial/` pages
      - Screenshot 2024-03-06 at 11 50 21 AM
- filtered out broken content links for platforms where links were not applicable
  - `/pages/[platform]/tools/console/adminui/start/index.mdx`
     - [prod](https://docs.amplify.aws/swift/tools/console/adminui/start/#:~:text=new%20browser%20tab-,Next%20steps,-You%20are%20ready)
     - [staging](https://correct-content-filtering.d1egzztxsxq9xz.amplifyapp.com/swift/tools/console/adminui/start/#next-steps)
  - `/pages/[platform]/tools/console/data/data-model/index.mdx`
  - /react-native/prev/build-a-backend/graphqlapi/upgrade-guide/ 
- removed outdated content with broken links from fragments
   - `/fragments/sdk/storage/android/getting-started.mdx`
      - [prod](https://docs.amplify.aws/swift/sdk/storage/getting-started/#mocking-and-local-testing)
      - [staging](https://correct-content-filtering.d1egzztxsxq9xz.amplifyapp.com/swift/sdk/storage/getting-started/) (still building)
   - `/fragments/sdk/storage/ios/getting-started.mdx`
   - `/fragments/sdk/api/ios/graphql.mdx`
   - `/fragments/sdk/api/android/graphql.mdx`

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] Swift
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://docs.amplify.aws/)` 
            HTML: `<a href="https://docs.amplify.aws/">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/fragments/sdk/storage/ios/getting-started.mdx [hunk 1]
```diff
@@ -86,7 +86,3 @@ Run `pod install --repo-update` before you continue.
    ```swift
    import AWSS3
    ```
-
-### Mocking and Local Testing
```
Comment (reesscot) [inline src/fragments/sdk/storage/ios/getting-started.mdx:90]: Guessing that the staging site is still building? Still seeing it here: https://correct-content-filtering.d1egzztxsxq9xz.amplifyapp.com/swift/sdk/storage/getting-started/#mocking-and-local-testing

**Review by reesscot — DISMISSED** [review]
LGTM, other than comment on staging site

File: src/fragments/sdk/storage/ios/getting-started.mdx (hunk 1 above)
Comment (katiegoines) [reply src/fragments/sdk/storage/ios/getting-started.mdx:90]: Yes, just added a note in the description.

Comment (reesscot) [reply src/fragments/sdk/storage/ios/getting-started.mdx:90]: I see it's gone now!

**Review by reesscot — APPROVED** [review]

**Review by dbanksdesign — APPROVED** [review]

**Review by renebrandel — APPROVED** [review]

**Review by hdworld11 — APPROVED** [review]

**Pull request merged**